### DISCUSSION TITLE
What the game really needs

### DISCUSSION THREAD

**Discussion by poteitouweb3** [sunflower-land/sunflower-land#3426] [Ideas]
I have a somewhat different view of web 3.0 games than people generally do.

In my opinion, the function of using blockchain technologies (such as NFT/SFT and Tokens) is to guarantee that the player actually owns the assets he has obtained and also to allow a black market to be legalized.

If you go to a website called "eldorado.gg" you will see something that should be obvious to all of us by now: Web 2.0 games have a GIANT illegal market. That's because that's how the economy works: Where there is supply, there will be demand, regardless of whether it is illegal or not. In this sense, we have to ask ourselves: Why do big games like CSGO, Valorant, Fortnite, Minecraft and others have such a vast and vivid parallel economy if these games don't even support their players in commercializing their assets? Well, the answer is simple: Entertainment.

The reason a CSGO player buys a $10,000 skin is that the game has a high dopamine value for him, which means that that $10,000 turns into hours and hours of justified pleasure (within that individual's perception of value). What happens in web 3.0 games is that people wonder what can be done to increase the value of assets, burn more resources and so on, but they never ask themselves: Who is willing to make the sacrifices so that value flows from one hand to the other?

Instead of asking whether Sunflower Land's features are burning enough resources, whether halvening is good or bad or what we should do to burn more resources, we should ask ourselves: Is the game fun enough for thousands of people to be willing to spend money without worrying about "making a profit" or "getting their money back"?

Nowadays one of the things I hear most is: "Who's going to play and spend block bucks for months without farming anything?" Completely disregarding the "fun" factor in this criticism.

I understand that the team is small and the budget is short and what is being built is VERY GOOD compared to these two factors, but we need to keep in mind that each gameplay mechanic becomes ammunition for content creators to generate more attractive content precisely for the majority audience of web 2.0.

If I'm not mistaken and the team is thinking along similar lines to me, I can imagine that the community island is the feature that will guarantee several mini games and even larger projects WITHIN Sunflower Land as a living ecosystem for game creation. In this sense, I have a few questions:

- How will marketing work to attract game creators to Sunflower Land?

- Will the development toolkit being developed be complete enough for major features such as combat systems and animations when?

- To ensure that the community island proposal grows quickly and successfully, does the team intend to seek investment rounds to expand the team and speed up deliveries?

**Comment by TheDcol** [comment]
> ..we should ask ourselves: Is the game fun enough for thousands of people to be willing to spend money without worrying about "making a profit" or "getting their money back"?

I always stated and still do that the priority of the game is to have engaging gameplay, be fun to play and have long-term progression. To reach a point where it becomes truly a game where crypto and non-crypto people would play it regardless of token/item prices. So in a way I think we are in agreement as to what the focus and aim is? (Unless I misunderstood)

![image](https://github.com/sunflower-land/sunflower-land/assets/108110065/1b4294d7-8ce3-415d-ab38-52d5508341f7)

**Comment by adamhannigan** [comment]
Thanks for the writeup. Some great points! 

**How will marketing work to attract game creators to Sunflower Land?**

The initiative is still early, but we have a few ideas lined up for this:

1. Partner with Polygon Game maker events. They sponsor Web3 hackathons deeply and are willing to collaborate on a Sunflower Land themed one. 
2. Partner with Phaser or Colyseus (technologies we use). No partnership progress has been made on this - we will be looking at some paid/affiliate sponsorship to encourage Web2 minigame builders

**Will the development toolkit being developed be complete enough for major features such as combat systems and animations when?**

Yes. Players can build anything they want - we provide the basic boilerplates and easy to use assets, but anyone can build anything (also in their own preferred Web language if they wish)

**To ensure that the community island proposal grows quickly and successfully, does the team intend to seek investment rounds to expand the team and speed up deliveries?**

Potentially. Based on the success of the initially rolled out portals will determine this decision. We've got enough capital to complete the initiative and hire extra if needed. Investment would only be needed for time sensitive or partnered opportunities

**Reply by poteitouweb3** [reply]
Awesome Adam! Thanks for the answers. Feeling bullish about Portals/Community Islands
